tests/models/hydrology/test_hydrology_model.py

```diff
@@ -99,7 +100,8 @@ def test_hydrology_model_initialization(
         assert repr(model) == "HydrologyModel(update_interval=1209600 seconds)"
         assert model.initial_soil_moisture == ini_soil_moisture
         assert model.initial_groundwater_saturation == ini_groundwater_sat
-        assert model.drainage_map == {0: [], 1: [0], 2: [1, 2]}
+        # VIVI - not sure about the value below.
```

Comment:
This will depend on the elevation input

Reply:
Yup. Which is that grid of `[[200,100],[10,10]]`. We probably just leave this - it passes tests - and then generate some more expansive tests of the drainage mapping system elsewhere. If that changes the implementation, we just come back and fix this value.